x-pack/test/security_solution_endpoint/apps/endpoint/resolver.ts

```diff
@@ -262,11 +262,21 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
         await esArchiver.load('endpoint/resolver_tree/library_events', { useCreate: true });
         await queryBar.setQuery('');
         await queryBar.submitQuery();
-        const expectedLibraryData = ['329 network', '1 library', '1 library'];
+        const expectedLibraryData = [
+          '1 authentication',
+          '1 session',
+          '329 network',
+          '1 library',
+          '1 library',
```

Comment:
Should there be 2 separate library events in this list?

Reply:
What you mean
Yes this is what it looks like